site2/website-next/docs/reference-configuration.md

```diff
@@ -164,6 +164,14 @@ Pulsar brokers are responsible for handling incoming messages from producers, di
 |brokerDeduplicationMaxNumberOfProducers| The maximum number of producers for which information will be stored for deduplication purposes.  |10000|
 |brokerDeduplicationEntriesInterval|  The number of entries after which a deduplication informational snapshot is taken. A larger interval will lead to fewer snapshots being taken, though this would also lengthen the topic recovery time (the time required for entries published after the snapshot to be replayed). |1000|
 |brokerDeduplicationProducerInactivityTimeoutMinutes| The time of inactivity (in minutes) after which the broker will discard deduplication information related to a disconnected producer. |360|
+|dispatchThrottlingRateInMsg| Default messages (per second) dispatch throttling-limit for whole broker. When the value is set to 0, default message dispatch throttling-limit is disabled. |0|
```

Comment:
same comments for all modifications in `website-next`

Reply:
done